### ISSUE TITLE
[Bug] vanish keybind

### ISSUE THREAD

**Issue by Yuujii-Alex** [Enubia/ghost-chat#832]
cannot change the vanish keybind, does nothing, no input

**Comment by Enubia** [comment]
Mind letting me know what exactly you tried? Did it just not recognize a specific key combination?

**Comment by Enubia** [comment]
Oh ok I'll look into this

**Comment by ChromesDuzez** [comment]
I am having the same issue here but the log shows no errors as to why I can't set a keybinding
```
[2024-04-02 16:17:48.625] [info]  App starting
[2024-04-02 16:17:48.642] [info]  Store created
[2024-04-02 16:17:48.695] [info]  Building overlay window
[2024-04-02 16:17:48.696] [info]  Overlay window state {
  x: 0,
  y: 0,
  width: 400,
  height: 600,
  isTransparent: false,
  isClickThrough: false,
  theme: null
}
[2024-04-02 16:17:48.826] [info]  Checking for update
[2024-04-02 16:17:48.845] [info]  Generated new staging user ID: 3218fe71-3f17-5772-b670-53688410003e
[2024-04-02 16:17:49.802] [info]  Update for version 2.9.0 is not available (latest version: 2.9.0, downgrade is disallowed).
[2024-04-02 16:17:49.804] [info]  [aA] update-not-available 2.9.0
[2024-04-02 16:18:06.148] [info]  Rerendering child
[2024-04-02 16:18:17.855] [info]  Vanishing overlay
[2024-04-02 16:18:17.856] [info]  Saved window state { x: 760, y: 220, width: 751, height: 420 }
[2024-04-02 16:20:56.693] [info]  Disabling Vanish
[2024-04-02 16:21:01.377] [info]  Opening config folder
[2024-04-02 16:21:34.644] [info]  Opening settings window
[2024-04-02 16:21:34.644] [info]  Creating settings window
[2024-04-02 16:21:34.645] [info]  Building settings window
[2024-04-02 16:21:34.645] [info]  Settings window state { x: 0, y: 0, width: 900, height: 760, theme: 'dark' }
[2024-04-02 16:21:43.763] [info]  Rerendering parent
[2024-04-02 16:21:48.517] [info]  Rerendering parent
[2024-04-02 16:24:09.290] [info]  Opening logs folder

```
